src/Compilers/CSharp/Test/Semantic/Semantics/InheritanceBindingTests.cs

```diff
@@ -4967,9 +4966,9 @@ class Derived : Base<int>
     public override void Method(int @in, ref int @ref) { }
 }
 ";
-            CompileAndVerifyDiagnostics(text, new ErrorDescription[] {
-                new ErrorDescription { Code = (int)ErrorCode.WRN_MultipleRuntimeOverrideMatches, Line = 5, Column = 25, IsWarning = true }, //C# can distinguish, but runtime can't
```

Comment:
>WRN_MultipleRuntimeOverrideMatches [](start = 61, length = 34)

Same comment as for the other tests affected by the removal of WRN_MultipleRuntimeOverrideMatches

Reply:
That is tested in `CodeGenOverridingAndHiding`.

---
In reply to: [442521581](https://github.com/dotnet/roslyn/pull/44025#discussion_r442521581) [](ancestors = 442521581)